experimental/dds/tree2/src/feature-libraries/contextuallyTyped.ts

```diff
@@ -234,6 +230,16 @@ export type ContextuallyTypedNodeData =
  */
 export type ContextuallyTypedFieldData = ContextuallyTypedNodeData | undefined;
 
+/**
+ * context object providing the typeSet and SchemaData and optional globalFieldKeySymbol to provide context to the data
+ * @alpha
+ */
+export interface ContextObject {
+	schemaData: SchemaDataAndPolicy;
+	typeSet: TreeTypeSet;
```

Comment:
All three of these fields are mutable and undocumented.
 Unless there is a clear reason (from the name or documentation) for what mutating them means and why it should be allowed, these should be read-only (and probably also documented). If read-only and with really useful field names and we stop package exporting it, then docs here wouldn't be as necessary.